design/www/app/chatt.route.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+    'use strict';
+
+angular.module('starter', ['ionic', 'ngStorage', 'ngResource', 'GoogleLoginService', 'facebookLoginService', 'ngMessages'])
+
+.config(function($stateProvider, $urlRouterProvider) {
+  $stateProvider
+
+    .state('login', {
+    url: '/login',
+    cache:false,
+    templateUrl: 'app/login/login.html',
+    controller: 'loginController',
+    controllerAs: 'vm'
```

Comment:
dont use 'vm' here. that was just an example. user proper controller name e.g 'login'